File: app/src/main/java/org/mifos/mobilebanking/models/accounts/savings/SavingsAccountWithdrawPayload.java
package org.mifos.mobilebanking.models.accounts.savings;

/*
 * Created by saksham on 02/July/2018
 */

public class SavingsAccountWithdrawPayload {

    String locale = "en";
    String dateFormat = "dd MMMM yyyy";
    String withdrawnOnDate;
    String note;

    public String getLocale() {
        return locale;
    }

    public void setLocale(String locale) {
        this.locale = locale;
    }

    public String getDateFormat() {
        return dateFormat;
    }

    public void setDateFormat(String dateFormat) {
        this.dateFormat = dateFormat;
    }

    public String getWithdrawnOnDate() {
        return withdrawnOnDate;
    }

    public void setWithdrawnOnDate(String withdrawnOnDate) {
        this.withdrawnOnDate = withdrawnOnDate;
    }

    public String getNote() {
        return note;
    }

    public void setNote(String note) {
        this.note = note;
    }
}

File: app/src/main/java/org/mifos/mobilebanking/models/accounts/savings/SavingsAccountWithdrawPayload.kt
package org.mifos.mobilebanking.models.accounts.savings

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

/*
 * Created by saksham on 02/July/2018
 */

@Parcelize
data class SavingsAccountWithdrawPayload (
    var locale: String = "en",
    var dateFormat: String = "dd MMMM yyyy",
    var withdrawnOnDate: String? = null,
    var note: String? = null
) : Parcelable
